src/NetEscapades.EnumGenerators/EnumToGenerate.cs

```diff
@@ -8,15 +8,19 @@ namespace NetEscapades.EnumGenerators;
     public readonly bool IsPublic;
     public readonly bool HasFlags;
     public readonly string UnderlyingType;
-    public readonly List<KeyValuePair<string, object>> Values;
+
+    /// <summary>
+    /// Key is the enum name. Value is the custom name setted by the <c>[Display(Name)]</c> attribute.
+    /// </summary>
+    public readonly List<KeyValuePair<string, string?>> Values;
```

Comment:
I think we should give this a different name? It's not the "values" now, it's the `Names`?

Reply:
Yes, ´Names´ much better.